aws/resource_aws_organizations_organization_test.go

```diff
@@ -94,17 +94,15 @@ func testAccAwsOrganizationsOrganization_AwsServiceAccessPrincipals(t *testing.T
 	})
 }
 
-func testAccAwsOrganizationsOrganization_EnabledPolicyTypes(t *testing.T) {
-	var organization organizations.Organization
+func addTestStepsAwsOrganizationsOrganization_EnabledPolicyTypes(policyTypes *[]string) *[]resource.TestStep {
```

Comment:
For consistency with the rest of the codebase, can you please leave this testing with its original style and just change the second `TestStep` to use the new value? We prefer having `TestStep` declared explicitly in test functions. Thanks.

Reply:
No problem!